frontend/components/inbox/TicketOverview.vue

```diff
@@ -125,7 +128,6 @@ export default {
 
       axios.get(`/api${window.location.pathname}`, {
         params: {
-          columns: true,
```

Comment:
Gaat dit nog wel goed met de list view in vue?

Reply:
Dit wordt in de toggleView functie afgehandeld. Niet in deze api